test/jit/call_complex.txt

```diff
@@ -0,0 +1,301 @@
+;;; TOOL: run-interp-minimal
+(module
+  ;; TODO: Find a better way to prevent JIT compilation than using memory.size
+  ;; TODO: Find a way to avoid baking in function offsets
```

Comment:
Opening an issue for both of these seems appropriate.

Reply:
Opened #158 for the first `TODO`. The 2nd was accidentally copied from another test and actually doesn't apply at all here; I've opened #159 for the original issue nonetheless and I'll go ahead and remove the `TODO` from this test.